### DISCUSSION TITLE
Collabora not working on nextcloud

### DISCUSSION THREAD

**Discussion by isZYKerman** [CollaboraOnline/online#10256] [Q&A]
Hi everyone,
I run CODE and nextcloud on a TrueNAS 24.10 device, in which the Apps are migrated from k3s to docker-compose. I cannot obtain a public IPv4, so I'm all on IPv6. This results in that I can't simply follow the tutorial to bind the two services with my domain, for apps in TrueNAS do NOT support IPv6. Therefore I used `docker network create office` and connected both containers to it and in the nextcloud administration panel, I used `http://ix-collabora-1-collabora-1:9980` as the server location, and it seemed to work ↓
![image](https://github.com/user-attachments/assets/7330c080-f1bb-471f-a476-3e0e2ef7ea77)
But when I open any file an error will occur ↓
![image](https://github.com/user-attachments/assets/3c99a0ba-7949-4645-a34e-0e5b03ad5dd2)
Do you have any solutions for this?

I see that 
![image](https://github.com/user-attachments/assets/8e845151-5b56-4ca6-a5e6-d0a14bad182f)
This is not the in-docker network domain and it is IPv6 only. Is that why the communication failed? If so, are there any solutions?

**Comment by vmiklos** [comment]
If you don't get an answer here, please try asking at https://forum.collaboraonline.com/

**Reply by isZYKerman** [reply]
OK. Thanks!
